src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/controller/TasksController.kt

```diff
@@ -440,9 +442,26 @@ class TasksController(
       )
 
     val tasks: MutableList<Task> = ArrayList()
-    async { tasks.addAll(assessmentTasks) }
-    async { tasks.addAll(placementRequestTasks) }
-    async { tasks.addAll(placementApplicationTasks) }
+    tasks.addAll(assessmentTasks)
+    tasks.addAll(placementRequestTasks)
+    tasks.addAll(placementApplicationTasks)
```

Comment:
Don't quote me on this, but you might be able to retain the behaviour you had before with:

```
val jobs = listOf(
  async { tasks.addAll(assessmentTasks) }
  async { tasks.addAll(placementRequestTasks) }
  async { tasks.addAll(placementApplicationTasks) }
)
jobs.awaitAll()
```

That way we can build all the tasks up asynchronously and not lose the performance benefits we had before.